src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderActivitiesResource.java

```diff
@@ -15,38 +17,37 @@
-            @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class, responseContainer = "List"),
-            @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class, responseContainer = "List")})
-    @ApiOperation(value = "The work activities that this offender is currently allocated to", notes = "This includes suspended activities", nickname = "getCurrentWorkActivities")
-    @GetMapping("/{offenderNo}/current-work")
-    public OffenderActivities getCurrentWorkActivities(@PathVariable("offenderNo") @ApiParam(value = "The offenderNo of the prisoner", required = true) final String offenderNo) {
-        return activitesService.getCurrentWorkActivities(offenderNo);
+    public OffenderActivitiesResource(final OffenderActivitiesService activitiesService) {
+        this.activitiesService = activitiesService;
     }
 
     @ApiResponses({
         @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class, responseContainer = "List")})
     @ApiOperation(value = "The work activities that this offender has been allocated to", notes = "This includes suspended activities", nickname = "getRecentStartedWorkActivities")
-    @GetMapping("/{offenderNo}/work-history")
-    public OffenderActivities getRecentStartedWorkActivities(@PathVariable("offenderNo") @ApiParam(value = "The offenderNo of the prisoner", required = true) final String offenderNo,
-            @RequestParam(value = "earliestEndDate") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @ApiParam(value = "Only include activities that have not ended or have an end date after the given date", example = "1970-01-02") final LocalDate earliestEndDate) {
-        return activitesService.getStartedWorkActivities(offenderNo, earliestEndDate);
+    @GetMapping("/{offenderNo}/activities-history")
+    public Page<OffenderActivitySummary> getRecentStartedActivities(@PathVariable("offenderNo") @ApiParam(value = "The offenderNo of the prisoner", required = true) final String offenderNo,
+                                                                        @RequestParam(value = "earliestEndDate") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @ApiParam(value = "Only include activities that have not ended or have an end date after the given date", example = "1970-01-02") final LocalDate earliestEndDate,
+                                                                        @RequestParam(value = "page", required = false) @ApiParam(value = "Target page number, zero being the first page", defaultValue = "0") final Integer pageIndex,
+                                                                        @RequestParam(value = "size", required = false) @ApiParam(value = "The number of results per page", defaultValue = "20") final Integer pageSize) {
+        final var pageIndexValue = ofNullable(pageIndex).orElse(0);
```

Comment:
this seems a bit naff - would be better to add a `defaultValue` onto `RequestParam`?

Reply:
ah probably, cheers. tbh I just copied this from another endpoint without looking too closely 🤦 ! Will change in the next one